web/src/components/PendingRequests.js

```diff
@@ -0,0 +1,47 @@
+import axios from "../axios";
+import { Box, Text } from "rebass"
+import { useEffect, useState } from "react"
+
+export const PendingRequests = ({username, organization}) => {
+    const [fetched, setFetched] = useState(false)
+    const [loading, setLoading] = useState(false)
+    const [error, setError] = useState(null)
+    const [pendingRequests,  setPendingRequests] = useState(null)
+
+    useEffect(() => {
+        if(!fetched && !error) {
+            setLoading(true)
+            axios.get(`/users/pending-invitations`).then((res) => {
+                setPendingRequests(res.data) 
+                setFetched(true);
+            })
+            .catch((e) => {
+                setError(e.message)
+            })
+            .finally(() => {
+                setLoading(false)
+            })
```

Comment:
this looks good. A later improvement would be a way for a user to acknowledge errors and dismiss them so that the server could be re-requested for the resources.